prereise/gather/griddata/hifld/const.py

```diff
@@ -438,6 +511,18 @@
     "west": -125,
 }
 
+proxy_substations = [
+    {"LATITUDE": 35.0514, "LONGITUDE": -81.0694, "NAME": "Catawba", "STATE": "SC"},
+    {
+        "LATITUDE": 44.2853,
+        "LONGITUDE": -105.3826,
+        "NAME": "Neil Simpson II",
+        "STATE": "WY",
+    },
+]
```

Comment:
What are these substations used for?

Reply:
Sometimes there are short transmission lines for which one endpoint seems to connect directly to a generator, but there's no substation there. In this case, that end of the line gets connected to the next closest substation, which sometimes modifies the transmission network topology and creates infeasibilities (i.e. a power plant with inflexible generation gets mapped to a low-capacity line which can't support the minimum power being pushed.

By adding proxy substations at the endpoints of these lines, we avoid making these connections, and the generator pushes its power only to the high-voltage network to which it's really connected to.